test/unittest/unittest_blas_kernels_cl.cpp

```diff
@@ -44,7 +44,7 @@ TEST(blas_kernels, dotCL_sgemv) {
   int width = 768;
 
   int height_b = 768;
-  int width_b = 96000;
+  int width_b = 2048;
```

Comment:
is there any particular reason to change this?

Reply:
No reason as such. Was testing with different values. Might have changed it to run this test faster to check other test results quickly.